frontend/tests/ui/conftest.py

```diff
@@ -252,7 +252,7 @@ def login_admin(self):
             yield
 
     @contextlib.contextmanager
-    def login(self, username, password):
+    def login(self, username, password, is_main_user_identity=True):
```

Comment:
```suggestion
    def login(self, username, password, *, is_main_user_identity=True):
```

así se evita la tentación de pasarle un `False` sin indicar a qué se refiere.

Reply:
Listo.